src/components/data_comps_nuopc/datm/src/atm_comp_nuopc.F90

```diff
@@ -886,6 +887,7 @@ subroutine datm_comp_advertise(importState, exportState, rc)
     call dshr_fldList_add(fldsExport, 'Sa_pbot'    )
     call dshr_fldList_add(fldsExport, 'Sa_shum'    )
     call dshr_fldList_add(fldsExport, 'Faxa_lwdn'  )
+    call dshr_fldList_add(fldsExport, 'Faxa_swdn'  )
```

Comment:
So we are now adding a new field Faxa_swdn that we are always planning to send ? However, none of the previous data modes included this. Have you verified that this still works with any of the other data modes (like DATM%NYF?)

Reply:
I think that it could be nice to have this flexibility for the regional applications because their configuration varies and requires different type of field. The current implemnetation uses CORE2 NYF and i'll run CESM full test suite as a part of the CMEPS PR. I'll also update this thread about the result of the CESM test.

Reply:
I am not sure why but if I run full test suite with following command

```
qcmd -l walltime=4:00:00 -- ./create_test --xml-testlist ../src/drivers/nuopc/cime_config/testdefs/testlist_drv.xml --xml-machine cheyenne --xml-category nuopc --compare apr26 --baseline-root /glade/p/cesmdata/cseg/nuopc_baselines
```

I am getting lots of difference. You could see the log using ```cs.status.20200601_154852_x7oya8```

Reply:
Maybe I am using wrong baseline. I download CESM app (**app_cesm-cmeps** branch), then I merge CIME master with my branch (**ufs-hafs-app_updated_v2**) and I am using CMEPS with #b0dc4cf and last commit is 

```
commit b0dc4cf7e3fcc65bafa6089e6bddb5fbecf2f2b0
Merge: 7ba0be9 61f4995
Author: mvertens <mvertens@users.noreply.github.com>
Date:   Thu May 28 15:49:04 2020 -0600

    Merge pull request #66 from DeniseWorthen/feature/ww3yaml
    
    ww3 additions to fd_nems.yaml
```

Reply:
After adding Faxa_swdn to esmFldsExchange_cesm_mod.F90 in CMEPS side, the A compset is fixed (X was also working before) and return PASS like following. 

```
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel CREATE_NEWCASE
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel XML
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel SETUP
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel SHAREDLIB_BUILD time=5
FAIL ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel NLCOMP
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel MODEL_BUILD time=20
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel SUBMIT
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel RUN time=29
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel COMPARE_base_rest
FAIL ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel BASELINE apr26: FIELDLIST field lists differ (otherwise bit-for-bit)
FAIL ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel TPUTCOMP Error: Computation time increase > 25 pct from baseline
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel MEMLEAK insuffiencient data for memleak test
PASS ERS_Vnuopc_Ln9_N3.f19_g17_rx1.A.cheyenne_intel SHORT_TERM_ARCHIVER
```

At this point message like **FIELDLIST field lists differ (otherwise bit-for-bit)** will be fine because the history file will have new field. I'll also look at other failures that leads to answer changes.

Reply:
After I use CESM repository nuopc_dev branch (417d6fa) all tests are passed except namelist changes.